rom-hacking/scripting/animation-commands/ac-index-anime.md

```diff
@@ -0,0 +1,97 @@
+# (870) _AC_INDEX_ANIME
+
+## Effect
+
+Animates the entity based on the given animation. The list of animations can be found at the bottom of this page.
+
+If the **time** argument is provided, the animation takes as long as specified.
+
+## Syntax
+
+```c
+_AC_INDEX_ANIME(animation, time?)
+```
+
+| Argument | Description | Types | Required |
+| - | - | - | - |
+| **animation** | The animation to perform | Float | Required |
+| **time** | The amount of time to animate, in seconds | Float | Optional |
```

Comment:
Are we sure this isn't frames?

Reply:
According to Smad this is in seconds https://github.com/TeamLumi/luminescent-team/pull/84#discussion_r1379018206